src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -129,3 +130,4 @@ def load_arguments(self, _):
 
     with self.argument_context('container attach') as c:
         c.argument('container_name', help='The container to attach to. If omitted, the first container in the container group will be chosen')
+        c.argument('no_move_cursor', options_list=['--no-move-cursor'], help='Do not move cursor up, useful when piping', action='store_true')
```

Comment:
Instead of duplicating this line, you should register it once under the `container` scope.

Reply:
But this only applies to `attach` and `logs`, not other command, is that the right way?

Reply:
Yes, it is. Since those commands don't have `no_move_cursor`, then this metadata would be ignored.